docs/3.0/field-options.md

```diff
@@ -350,11 +350,11 @@ Related:
 
 ## Align text on Index view
 
-It's customary on tables to align numbers to the right. You can do that using the `index_text_align` option. Valid values are `:right` or `:center`.
+It's customary on tables to align numbers to the right. You can do that using the `html` option.
 
 ```ruby{2}
 class ProjectResource < Avo::BaseResource
-  field :users_required, as: :number, index_text_align: :right
+  field :users_required, as: :number, html: {index: {wrapper: {classes: "text-right"}}}
```

Comment:
This is even better. I was thinking of suggesting a CSS selector 

_*Incorrect selector, but just to get the point across 👇_
```css
[data-resource-name="TeamResource"] [data-field-type="external_image"][data-field-id="logo"] [data-slot="label"]{
  text-align: left;
}
```

Reply:
We must ensure we have all the [text-align classes](https://tailwindcss.com/docs/text-align) whitelisted.
Can you please add them to the [whitelist generator](https://github.com/avo-hq/avo-3/blob/main/scripts/export-tailwind-safelist.js#L23)? See the `ignoredButtonClasses.push()` example to do the same.

Reply:
Good catch! Totally forgot about that